### DISCUSSION TITLE
use remix with ui framework

### DISCUSSION THREAD

**Discussion by s446683009** [remix-run/remix#5380] [Q&A]
I am a beginner of remix. I want to ask if remix can use  ui framework? Because the  ui framework contains a lot of createContext.it used in brower,but I don't know how to use ti with remax.
such as material ui.

**Comment by jrestall** [comment]
Official Mui Remix example:
https://github.com/mui/material-ui/tree/master/examples/remix-with-typescript

Remix examples repo with a lot of examples of other ui libs like mantine/chakra
https://github.com/remix-run/examples

Personally I like shadcn since it uses simple tailwindcss and headless Radix UI, so no need for CSS-in-JS, but there's no Remix example yet so probably not a good option for a beginner.
https://ui.shadcn.com/

**Reply by s446683009** [reply]
Thank you very much for your answer.
